src/ripple/overlay/impl/Manifest.cpp

```diff
@@ -206,6 +205,25 @@ ManifestCache::configManifest (
     }
 }
 
+boost::optional<std::string>
+ManifestCache::member (PublicKey const& identity) const
```

Comment:
I think it was meant to be symmetric with the `ValidatorList::member` function which does this - the reasoning with that case is that it can help avoid two calls to `find` from a hot path. I don't think it applies here.